ppxlib.opam

```diff
@@ -14,7 +14,14 @@ run-test: [
   ["dune" "runtest" "-p" name "-j" jobs] { ocaml:version >= "4.07" }
 ]
 depends: [
-  "ocaml"                   {>= "4.04.1" & < "4.13"}
+  "ocaml" {>= "4.04.1" & < "4.13"}
+  (
+    ("ocaml" {>= "4.04.1" & < "4.07"}) |
+    (
+      "ocaml" {>= "4.07" & < "4.13"}
+      "cinaps" {with-test & >= "v0.12.1"}
+    )
+  )
```

Comment:
Note on that: If this is merged, I'd rather keep it in the repository and not put it in the releases as the opam solver struggles quite a bit with disjunctions.

Reply:
Hopefully we'll rather count cinaps releasing a new 4.04 compatible version. See https://github.com/ocaml-ppx/cinaps/pull/4.

Out of curiosity, how would you handle it in the release? Just dropping the test deps on cinaps? That doesn't feel right.

Reply:
It didn’t seem to be too bad when I tested it locally?